src/main/java/frc/robot/subsystems/Lift.java

```diff
@@ -33,9 +33,12 @@ public class Lift extends SubsystemBase {
   /** Overall position of the lift including both elevator and arm */
   public enum LiftPosition {
     GRAB_FROM_INTAKE,
-    SHELF,
```

Comment:
Shelf removed? In several places. We still need the shelf position (this is for the loading station)